.spice/datasets/prices.yaml

```diff
@@ -0,0 +1,13 @@
+name: spiceai.datasets.prices
+type: append
+access:
+  read:
+    - "*"
+migrations:
+  - name: create_view
+    sql: |
+    CREATE OR REPLACE VIEW spiceai.datasets.prices AS (
+      (SELECT pair,price,timestamp_ms,asset_id0,asset_id1 FROM icedatalake.prices.trades )
```

Comment:
We don't include `exchange`,`tradeid` or `swap_route` in the simplified view. This might technically produce duplicate rows?